tutormfe/plugin.py

```diff
@@ -122,7 +121,11 @@ def _add_remote_mfe_image_iff_customized(images, user_config):
     )
 ):
     with open(path, encoding="utf-8") as patch_file:
-        tutor_hooks.Filters.ENV_PATCHES.add_item((os.path.basename(path), patch_file.read()))
+        if path.endswith('lms-production-settings') or path.endswith('lms-development-settings'):
+            #Here we need to load it with a high priorty?
+            tutor_hooks.Filters.ENV_PATCHES.add_item((os.path.basename(path), patch_file.read()))
+        else:
```

Comment:
So I did it in this commit https://github.com/overhangio/tutor-mfe/pull/69/commits/2b6065c764c4e55bb5aeec680c8798409c171fc5 , I got an error though because I guess this change is not relased in tutor 14.0.2, it's still in nightly but it will be there for tutor 15, right?

Reply:
Instead of setting the priority explicity to 1, I think we should import Tutor's suggested values and use them.  That will give downstream plugins the option of specifying even higher priorities if they want to.

Also, there's a way to keep this DRYer.

```
from tutor.hooks import priorities

priority = priorities.DEFAULT
if path.endswith('lms-production-settings') or path.endswith('lms-development-settings'):
    priority = priorities.HIGH

tutor_hooks.Filters.ENV_PATCHES.add_item((os.path.basename(path), patch_file.read()), priority=priority)
```

Reply:
good call. I would DRY it up even further and remove the substring check in favor of a more explicit string comparison:
```python
from tutor.hooks import priorities

priority = priorities.DEFAULT
patch_name = os.path.basename(path)
if patch_name in ['openedx-lms-production-settings', 'openedx-lms-development-settings']:
    priority = priorities.HIGH

tutor_hooks.Filters.ENV_PATCHES.add_item((patch_name, patch_file.read()), priority=priority)
```